src/site/content/en/react/how-amp-can-guarantee-fastness-in-your-nextjs-app/index.md

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
staged doc: https://deploy-preview-1705--web-dev-staging.netlify.com/how-amp-can-guarantee-fastness-in-your-nextjs-app